tools/build_variables.bzl

```diff
@@ -433,36 +433,39 @@ lazy_tensor_core_sources = [
     "torch/csrc/lazy/core/lazy_view.cpp",
     "torch/csrc/lazy/core/metrics.cpp",
     "torch/csrc/lazy/core/multi_wait.cpp",
+    "torch/csrc/lazy/core/ops/arithmetic_ir_ops.cpp",
+    "torch/csrc/lazy/core/ops/utils.cpp",
     "torch/csrc/lazy/core/permutation_util.cpp",
     "torch/csrc/lazy/core/shape.cpp",
     "torch/csrc/lazy/core/shape_inference.cpp",
     "torch/csrc/lazy/core/tensor.cpp",
     "torch/csrc/lazy/core/tensor_impl.cpp",
     "torch/csrc/lazy/core/tensor_util.cpp",
     "torch/csrc/lazy/core/thread_pool.cpp",
-    "torch/csrc/lazy/core/view_ops/as_strided.cpp",
-    "torch/csrc/lazy/core/view_ops/as_strided_view_update.cpp",
-    "torch/csrc/lazy/core/view_ops/diagonal.cpp",
-    "torch/csrc/lazy/core/view_ops/diagonal_view_update.cpp",
-    "torch/csrc/lazy/core/view_ops/narrow.cpp",
-    "torch/csrc/lazy/core/view_ops/narrow_view_update.cpp",
-    "torch/csrc/lazy/core/view_ops/permute.cpp",
-    "torch/csrc/lazy/core/view_ops/resize.cpp",
-    "torch/csrc/lazy/core/view_ops/select.cpp",
-    "torch/csrc/lazy/core/view_ops/squeeze.cpp",
-    "torch/csrc/lazy/core/view_ops/unsqueeze.cpp",
-    "torch/csrc/lazy/core/view_ops/select_view_update.cpp",
-    "torch/csrc/lazy/core/view_ops/view.cpp",
     # We should better segment the sources, but for now there are actually dependencies
     # from some core files on some of these ts_backend files
     # so we continue to build these parts of ts_backend in all build configs
     "torch/csrc/lazy/ts_backend/config.cpp",
-    "torch/csrc/lazy/ts_backend/ops/arithmetic_ir_ops.cpp",
+    "torch/csrc/lazy/ts_backend/dynamic_ir.cpp",
```

Comment:
> I believe it was because of a failure in one of the mobile builds. I can't see any CI history, but IIRC that was what it was. Though, if it works without that change, then I'm all for reverting it

hmm. well it seems more right to move it back to ts_backend srcs and then re-triage that mobile build i guess.

the specific error is an undefined symbol `torch::lazy::LowerTSBuiltin` when compiling dynamic_ir.cpp without compiling the rest of ts_backend which supplies the lowering stuff.

i'd like to improve the source segmentation too.. i forgot why i left so many of them in the core build in the first place.

Reply:
@antoniojkim actually does your IR builder finally make it possible to move all the ts_backend/ops/* files into the lazy_tensor_ts_sources list?  I think the reason we needed to compile them in core before was becuase we directly included/linked their symbols from core code.  If your PR adds a layer of indirection to core, it should be possible for core to only depend on the interface.